### PR TITLE
[now-dev] Fix flickering and improve message

### PR DESCRIPTION
**Pull request by thasophearak**
This PR fixed flickering and improve output message.

### CODE REVIEW

File: src/util/dev/server.ts [hunk 1]
```diff
@@ -490,14 +490,12 @@ export default class DevServer {
       }
     );
     if (needsInitialBuild.length > 0) {
-      const stopSpinner =  wait('Building...');
+      this.output.log('Creating initial build');
```
Comment (AndyBitz) [inline src/util/dev/server.ts]: Maybe we should make this `builds` instead of `build` when there is more than one build?

Comment (AndyBitz) [reply src/util/dev/server.ts]: We have a plural function that can take care of that

Comment (thasophearak) [reply src/util/dev/server.ts]: @AndyBitz Thank! I updated it.

**Review by AndyBitz — COMMENTED** [review]
Looks fine to me, but why did it start to flicker? Or did it always flicker?

Because, I think it would still be better to indicate that something is happening instead of just showing text.

**Comment by thasophearak** [comment]
@AndyBitz Originally it isn't spinning. `executeBuild` already have it. The flickering introduce in this PR https://github.com/zeit/now-cli/pull/2506/files#diff-b2618e59e1594d139e5d9df68ca62dc3R179

**Review by AndyBitz — APPROVED** [review]

**Pull request merged**